src/psyclone/tests/psyad/tl2ad_test.py

```diff
@@ -37,6 +37,9 @@
 within the psyad directory.
 
 '''
+
+# pylint: disable=protected-access
```

Comment:
You don't need this - it's set in tests/.pylintrc.

Reply:
OK, I'll remove them. The reason I've started adding them is because the `./pylintrc` defaults are only picked up if I run tests from the `test` directory, at least for my version of pylint, and I usually go to where the tests are. I guess I'll just have to remember to specify specific tests or directories from the `tests` directory in the future.